server.js

```diff
@@ -1,30 +1,44 @@
-import express from "express";
-import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
-
-// Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
-
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
-
-// Start the server
-app.listen(port, () => {
-  console.log(`Server running on http://localhost:${port}`);
-});
+import express, { response } from "express";
+import cors from "cors";
+
+// If you're using one of our datasets, uncomment the appropriate import below
+// to get started!
+// import avocadoSalesData from "./data/avocado-sales.json";
+import booksData from "./data/books.json";
+// import goldenGlobesData from "./data/golden-globes.json";
+// import netflixData from "./data/netflix-titles.json";
+// import topMusicData from "./data/top-music.json";
+
+// Defines the port the app will run on. Defaults to 8080, but can be overridden
+// when starting the server. Example command to overwrite PORT env variable value:
+// PORT=9000 npm start
+const port = process.env.PORT || 8080;
+const app = express();
+
+// Middlewares to enable cors and json body parsing
+app.use(cors());
+app.use(express.json());
+
+// My routes
+app.get("/", (req, res) => {
+  res.json({responseMessage: "Welcome to my page where you can find information on 500 books. Please add /books/ and then a number in between 1-500, or /languages/ and for example 'fre' to see books in french." }) // Expects an json object and will be sent accordingly. More secure. 
+});  
+
+app.get("/books/:bookID", (req, res) => {

```

Comment:
I noticed that some id:s isn't there and I can't load the json-file to see if they're missing in there. 
Weirdly there is no 5, 6 and 7...